xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/pdfheader.vm

```diff
@@ -1,6 +1,8 @@
 #set ($title = "$!{pdfdoc.display('title', 'rendered', '', $pdfobj)}")
-#if ($title == '')
+#if ($title == '' && $doc.space != $tdoc.plainTitle)
   $!{escapetool.xml($doc.space)} - $!{escapetool.xml($tdoc.plainTitle)}
+#elseif ($title == '')
+  $!{escapetool.xml($tdoc.plainTitle)}
 #else
```

Comment:
Fixed.
I will edit the PR description in order to add more details about this issue.